.changeset/tricky-dolphins-film.md

```diff
@@ -0,0 +1,5 @@
+---
+"@khanacademy/perseus": major
```

Comment:
Just a nit: I originally strongly agreed with `major`, but after looking at this I'd argue it's a `minor`. The outward API isn't changing, just how we're responding to the API (by not using part of it). When you remove `satStyling` from `APIOptions` completely I'd call it `major`...

As I typed this I realized this is probably what you're doing, just incrementally. Disregard, but I'll leave this note for other devs or future me.

Reply:
Your hunch is correct.  Since this is on a feature branch, I'm going to treat the entire feature as a single changeset.  I should maybe have done empty change sets until the last commit, but it's here now.  :)